hasher-matcher-actioner/webapp/src/pages/bank-management/ViewBankMember.tsx

```diff
@@ -121,18 +148,26 @@ export default function ViewBankMember(): JSX.Element {
           <Col md={{span: 6}}>
             <Container>
               <Row>
-                <Col xs={{span: 10}}>
+                <Col>
                   <h3>Member Details</h3>
                 </Col>
-                <Col xs={{span: 2}}>
-                  <Button
-                    size="sm"
-                    variant="secondary"
-                    onClick={() => setPollBuster(pollBuster + 1)}>
-                    Refresh
-                  </Button>
-                </Col>
               </Row>
+              {member.is_removed ? (
+                <Row>
+                  <Col>
+                    <Alert variant="warning">
+                      <Alert.Heading>
+                        Member is marked as removed!
+                      </Alert.Heading>
+                      <p>
+                        This member is marked as removed. Signals from this
```

Comment:
Nit: my next question as a user is how do I re-add this? and can I delete it completely (okay is we say 'feature pending')